src/ThreeEditor/js/Menubar.Add.js

```diff
@@ -52,25 +65,46 @@ function MenubarAdd(editor) {
 	options.add(createOption('option', strings.getKey('menubar/add/sphere'), () => {
 
 		var geometry = new THREE.SphereGeometry(1, 16, 8, 0, Math.PI * 2, 0, Math.PI);
-		var mesh = new THREE.Mesh(geometry, material);
+		var mesh = new THREE.Mesh(geometry, material.clone());
 		mesh.name = 'Sphere';
 
 		editor.execute(new AddObjectCommand(editor, mesh));
 	}));
 
+
+
+
+
 	// Cylinder
 	options.add(createOption('option', strings.getKey('menubar/add/cylinder'), () => {
 
 		var geometry = new THREE.CylinderGeometry(1, 1, 1, 16, 1, false, 0, Math.PI * 2);
 		geometry.translate(0, geometry.parameters.height / 2, 0);
-		var mesh = new THREE.Mesh(geometry, material);
+		var mesh = new THREE.Mesh(geometry, material.clone());
 		mesh.name = 'Cylinder';
-
 		editor.execute(new AddObjectCommand(editor, mesh));
 	}));
 
 	options.add(new UIHorizontalRule());
 
+	// HemisphereLight
+
+	options.add(createOption('option', strings.getKey('menubar/add/hemispherelight'), () => {
+
+		var skyColor = 0x00aaff;
+		var groundColor = 0xffaa00;
+		var intensity = 1;
+
+		var light = new THREE.HemisphereLight(skyColor, groundColor, intensity);
+		light.name = 'HemisphereLight';
+
+		light.position.set(0, 10, 0);
```

Comment:
Why [0,10,0] ?

Reply:
No reason, code copied from original Menubar.Add.js, to add any light.

Reply:
See https://github.com/yaptide/ui/pull/55#discussion_r707208362

Reply:
Add a notice from where the code was copied